### PR TITLE
guest_agent: Add function of get basic info of vm

### PR DESCRIPTION
**Pull request by xiagao**
Get host name, timezone and users info via qga command
ID: 1809416
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -903,3 +903,29 @@ def set_memory_blocks(self, mem_blocks_list):
         """
         cmd = "guest-set-memory-blocks"
         return self._cmd_args_update(cmd, mem_blks=mem_blocks_list)
+
+    def get_hostname(self):
+        """
+        Get host name of vm.
+        """
+        cmd = "guest-get-hostname"
```
Comment (anascko) [inline virttest/guest_agent.py]: @xiagao 
guest-get-host-name

Comment (xiagao) [reply virttest/guest_agent.py]: yes, modified, thanks.

**Comment by xiagao** [comment]
@luckyh  Call for your review, thanks.

**Comment by luckyh** [comment]
@xiagao would you please provide the test result? thanks!

**Comment by xiagao** [comment]
@luckyh 
 (1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_os_basic_info.q35: PASS (85.44 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_os_basic_info.q35: PASS (103.88 s)

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**